sections/sprint-1/prototypes.tex

```diff
@@ -30,7 +30,10 @@ \section{Prototypes}
 \end{figure}
 
 When all users have connected, the host can start the game, and they will now see the virtual game field, as seen on \autoref{fig:prototype:ingame}.
-In this prototype, the player's icon is highlighted by having a solid color, whereas the other players are just shown as outlines.
+In this prototype, the players' icons are represented by small squares and the goals are represented by larger rectangles.
```

Comment:
```suggestion
In this prototype, the players' icons are represented by small squares and the goals are represented by larger rectangles and have the same color as the players that they belong to.
```

Reply:
that have the same color as the players they belong to.

Reply:
😡